EpsilonCCS/Src/main.c

```diff
@@ -140,10 +142,12 @@ int main(void)
     /* USER CODE END RTOS_TIMERS */
 
     /* Create the thread(s) */
-    /* definition and creation of defaultTask */
     osThreadDef(canRxTask, parseCanTask, osPriorityNormal, 1, configMINIMAL_STACK_SIZE);
     parseCanHandle = osThreadCreate(osThread(canRxTask), NULL);
 
+    osThreadDef(telemetryOutTask, sendTelemetryTask, osPriorityNormal, 1, configMINIMAL_STACK_SIZE);
```

Comment:
This is probably fine as is currently but I think it is rather likely that the stack size will need to be increased as more code is added to this.